src/common/types.ts

```diff
@@ -993,3 +993,5 @@ export interface PointsCollection {
   symbol: string
   image: string
 }
+
+export type GamePageActions = 'install' | 'launch' | 'uninstall'
```

Comment:
a best practice would be to put this type in https://github.com/HyperPlay-Gaming/utils/tree/main/src to keep the code dry and share with the store or other repos that might need this type (e.g. summon web portal)

Reply:
apologies if I missed it, but I don't think we are using uninstall anywhere. Can we remove it if it is not implemented?

Reply:
Yeah, I thought on having it in case we want to implement uninstall from the store game page in the future. But I will remove it for now.

Reply:
Thanks for reminding me of the utlils lib, I added the type there and updated both prs.

Reply:
done and done